mercury_engine_data_structures/formats/bldef.py

```diff
@@ -0,0 +1,31 @@
+from construct import Construct, Container, Struct, Const, Int32ul, Terminated, Hex
+
+from mercury_engine_data_structures.formats import BaseResource
+from mercury_engine_data_structures.formats.property_enum import PropertyEnum
+from mercury_engine_data_structures.game_check import Game
+from mercury_engine_data_structures import type_lib
+
+BLDEF = Struct(
```

Comment:
Isn't this just a basic `standard_format.game_model('CLightManager', 0x02000001)`?

Reply:
No, its a weird format thing. The bldef uses a `CLightManager` crc with that, but instead of the normal `CLightManager` data it contains a field called `pLightManager` which contains the `CLightManager` shown in reflection. So if you try a standard format it tries to parse it as a `CLightManager` and doesn't expect the `pLightManager` field.

Reply:
have you tried the exact code dark suggested? I believe he's correct

Reply:
Yep, you're right. I know i'd tried `standard_format.create` before and it wouldn't work but `game_model` seems to. I'll throw up a new commit in a second.